styles.css

```
@@ -0,0 +1,121 @@
+:root {
+  --primary-color: #861a92; 
+  --secondary-color: #38b2ac; 
+  --text-color: #4a4a4a;
+  --bg-color: #f9f9f9;
+  --card-bg: #fdf6e3; 
+
+  --borfer-radius: 8px;
+  --spacing: 20px;
+  --max-width: 1200px
```

Ohne das geht nix ;)
```suggestion
  --max-width: 1200px;
```